google-cloud-core/src/main/java/com/google/cloud/GrpcTransportOptions.java

```diff
@@ -213,12 +213,13 @@ private GrpcTransportOptions(Builder builder) {
    * Returns a channel provider from the given default provider.
    */
   public static ChannelProvider setUpChannelProvider(
-      InstantiatingChannelProvider.Builder providerBuilder, ServiceOptions<?, ?> serviceOptions) {
+      InstantiatingChannelProvider.Builder providerBuilder,
+      ServiceOptions<?, ?> serviceOptions) {
```

Comment:
revert the line formatting - there are a few more line reformattings you can revert too.